src/plugins/interactive_setup/public/theme/utils.ts

```diff
@@ -6,14 +6,14 @@
  * Side Public License, v 1.
  */
 
-import type { EuiThemeColorMode } from '@elastic/eui/src/services/theme/types';
+import { COLOR_MODES_STANDARD } from '@elastic/eui';
+import type { EuiThemeColorModeStandard } from '@elastic/eui';
 
 import type { CoreTheme } from '../../../../core/public';
 
 /**
  * Copied from the `kibana_react` plugin, remove once https://github.com/elastic/kibana/issues/119204 is implemented.
  */
-export const getColorMode = (theme: CoreTheme): EuiThemeColorMode => {
-  // COLOR_MODES_STANDARD is not exported from eui
-  return theme.darkMode ? 'DARK' : 'LIGHT';
+export const getColorMode = (theme: CoreTheme): EuiThemeColorModeStandard => {
+  return theme.darkMode ? COLOR_MODES_STANDARD.dark : COLOR_MODES_STANDARD.light;
```

Comment:
🎉

Reply:
Pinging @pgayvallet, also, as some changes in the PR are relevant to #118866